test/e2e/capacitytest/capacity_test.go

```diff
@@ -1,4 +1,4 @@
-package e2e
+package capacitytest
```

Comment:
To keep this PR focused, I recommend opening a separate PR that restructures the existing tests.

Reply:
https://github.com/equinix/metal-cli/pull/336